cmake/Modules/LocalAom.cmake

```diff
@@ -145,7 +145,8 @@ else()
 
         foreach(_config_setting CMAKE_C_FLAGS CMAKE_CXX_FLAGS CMAKE_EXE_LINKER_FLAGS)
             foreach(_config_type DEBUG RELEASE MINSIZEREL RELWITHDEBINFO)
-                set(${_config_setting}_${_config_type} ${${_config_setting}_${_config_type}_ORIG} CACHE STRING "" FORCE)
+                unset(${_config_setting}_${_config_type} CACHE)
```

Comment:
It might be helpful to document these two loops as restoring the variables.

Reply:
Done.